apps/nuxt3-ssr/nuxt.config.ts

```diff
@@ -29,6 +29,11 @@ const config = {
   nitro: {
     compressPublicAssets: { brotli: true },
   },
+  imports: {
+    transform: {
+      exclude: [/\bmeta-data-utils\b/], // exclude from transform as this is already transformed by mono-repo build system
```

Comment:
ah, this is unfortunate.

Reply:
yea it surprised me , someone mentioned this in a issue and the nuxt guys sad that this is a convenience option to tranform mono repo files by default.